core/common/boot/src/test/java/org/eclipse/edc/boot/system/ExtensionLoaderTest.java

```diff
@@ -98,6 +102,24 @@ void loadMonitor_whenNoMonitorExtension() {
         assertTrue(monitor instanceof ConsoleMonitor);
     }
 
+    @ParameterizedTest
+    @MethodSource("provideLogLevelVariantArgs")
+    void loadMonitor_programArgsSetConsoleMonitorLogLevel(String programArgs, ConsoleMonitor.Level level) {
+
+        var monitor = ExtensionLoader.loadMonitor(new ArrayList<>(), programArgs);
+
+        assertThat(monitor).extracting("level").isEqualTo(level);
+    }
+
+    @ParameterizedTest
+    @MethodSource("provideLogLevelWrongArgs")
```

Comment:
pls use the `ArgumentsProvider` b/c it is strongly typed and more resilient w.r.t. to refactoring

Reply:
Is commit [aa0c624](https://github.com/eclipse-edc/Connector/pull/4476/commits/aa0c62454af93b2729374aa2dd0923e3c53fa1b9) ok? I can remove the last commit.